.github/workflows/test-integration-rosa.yaml

```diff
@@ -0,0 +1,352 @@
+name: "Test - Integration - ROSA"
+
+# description: This workflow perform integration tests against ROSA platform, it only prepares the cluster to run the tests
+
+on:
+  schedule:
+    - cron: '0 3 * * 1,3,4,6' # Runs at 3 AM on Monday, Wednesday, Thursday, and Saturday
```

Comment:
weird choice in schedule, or?
I'd have expected `1-5` or `1,3,5`.
I'm not sure Saturday will do much considering it's a weekend.

Reply:
I think they're using GMT, so even in the US it would be post work day.